app/services/requirements/document_checker.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module Requirements
+  class DocumentChecker
```

Comment:
Should this class have a concern about required tags? like organisations for instance?

Reply:
My intention here was just to centralise the existing requirements, rather than add new ones. We don't currently check for tag stuff, but maybe this should be a card. I think this would be in a different PR, anyway.